### DISCUSSION TITLE
Would be possible to add a few more dev subdomains to the License Enforcement list?

### DISCUSSION THREAD

**Discussion by leowebguy** [craftcms/cms#14588] [Ideas]
Would be possible to add rt, release to [this](https://craftcms.com/knowledge-base/how-craft-license-enforcement-works#how-do-we-determine-craft-is-running-on-a-public-domain
) list?

Thanks in advance!

**Comment by brandonkelly** [comment]
I’ve never heard of that one. What does it stand for? I’m not sure we’d want to add it since it seems like it could likely stand for a number of things that aren’t actually dev/testing-related.

**Comment by leowebguy** [comment]
Would be release test.

no worries. I'll switch to preprod or another one current in the list.

Thanks
